sgf-viewer/src/landing-page-content.ts

```diff
@@ -1,81 +1,59 @@
+        border: false,
     },
     {
-        image: "/images/far-logo-textless.svg",
-        imageName: "blog",
-        description: "Blog",
+        image: "/images/attack-blog.svg",
+        imageName: "Attack blog icon",
+        description: "Attack blog",
         url: "https://far.ai/post/2023-07-superhuman-go-ais/",
         color: "rgb(70,205,206,0.5)",
         border: false,
     },
-    // {
-    //     image: "/images/ICML-logo.svg",
-    //     imageName: "ICML",
-    //     description: "Poster",
-    //     url: "https://icml.cc/virtual/2023/poster/24524",
-    //     color: "rgb(35,31,32,0.5)",
-    // },
     {
         image: "/images/video-screenshot.jpg",
-        imageName: "video screenshot",
-        description: "Video",
-        url: "https://slideslive.com/39006680/adversarial-policies-beat-superhuman-go-ais",
+        imageName: "Attack video screenshot",
+        description: "Attack video",
+        url: "https://recorder-v3.slideslive.com/#/share?share=83403&s=1a8b2506-2198-4d30-a487-fdb05a497f5f",
         color: "rgb(31,24,42,0.5)",
         border: false,
     },
-    // {
-    //     image: "/images/pass-example.png",
-    //     imageName: "pass attack",
-    //     description: "View example games showcasing our pass attack.",
-    //     url: "/pass-based-attack"
-    // },
-    // {
-    //     image: "/images/non-transitive-figure.png",
-    //     imageName: "human evaluation",
-    //     description: "Read our evaluation of games demonstrating the non-transitive nature of the attacks.",
-    //     url: "/human-evaluation",
-    // },
-    // {
-    //     image: "/images/activation-analysis.png",
-    //     imageName: "activation analysis",
-    //     description: "Read our analysis of KataGo internal activations under the cyclic attack.",
-    //     url: "/activation-plots",
-    // },
-    // {
-    //     image: "/images/openreview_logo.png",
-    //     imageName: "open review",
-    //     description: "OpenReview",
-    //     url: "https://openreview.net/forum?id=Kyz1SaAcnd",
-    //     color: "rgb(140,27,18,0.5)",
-    // },
     {
         image: "/images/financial-times-logo.png",
-        imageName: "financial times",
-        description: "Press",
+        imageName: "Financial Times logo",
+        description: "Attack press",
         url: "https://www.ft.com/content/175e5314-a7f7-4741-a786-273219f433a1",
         color: "rgb(252,208,174,0.5)",
         border: false,
     },
+    {
+        image: "/images/defense-paper.svg",
+        imageName: "Defense paper icon",
+        description: "Defense paper",
+        url: "https://www.overleaf.com/project/654eac3d52dffeb1a092ab18",
+        color: "rgb(0,0,0,0.5)",
+        border: false,
+    },
+    {
+        image: "/images/defense-blog.svg",
+        imageName: "Defense blog icon",
+        description: "Defense blog",
+        url: "https://docs.google.com/document/d/1wcLUZLXejtuCHNCSPFfDVM0peLmV85046dmYsPeYhw4/edit",
```

Comment:
Final link will be: https://far.ai/post/2024-05-go-defense/ 
Current deployed preview link is: https://deploy-preview-131--alignmentfund.netlify.app/post/2024-05-go-defense/